packages/react-dom/src/events/DOMEventProperties.js

```diff
@@ -0,0 +1,175 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import type {EventPriority} from 'shared/ReactTypes';
+import type {
+  TopLevelType,
+  DOMTopLevelEventType,
+} from 'legacy-events/TopLevelEventTypes';
+import type {DispatchConfig} from 'legacy-events/ReactSyntheticEventType';
+
+import * as DOMTopLevelEventTypes from './DOMTopLevelEventTypes';
+import {
+  DiscreteEvent,
+  UserBlockingEvent,
+  ContinuousEvent,
+} from 'shared/ReactTypes';
+import {simpleEventPluiginEventTypes} from './SimpleEventPlugin';
```

Comment:
Typo: simpleEvent**Pluigin**EventTypes

But this file now imports the `SimpleEventPlugin` and creates a cyclic dependency between that and `DOMEventProperties`